src/WebJobs.Extensions.DurableTask/LinuxAppServiceFileLogger.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+#if !FUNCTIONS_V1
+using Mono.Unix.Native;
+#endif
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// The File logger for linux dedicated. Manages file rolling and is concurrency-safe.
```

Comment:
Can we add a comment where we copied this from, with some notes about how we changed this?

Reply:
Good point, will do

Reply:
Done!